docs/tutorials/extending_admin_ui/3b_adding_edit_button_content_list.md

```diff
@@ -0,0 +1,229 @@
+# Step 3b - Adding Edit button to content list
+
+In this step you will add an edit button to the content list. The following tutorial is based on the [Step 2 - Creating a content list](2_creating_a_content_list.md).
+
+## Modify the controller
+
+Introduce changes to `src/Controller/AllContentListController.php`, so that it allows content editing from the content list.
+
+First, add the new `FormFactory $formFactory` parameter and the `$this->formFactory = $formFactory;` argument to the `__construct` function:
```

Comment:
I think this is too simplified, and to much step by step.
I would go with something along 'Inject EzSystems\EzPlatformAdminUi\Form\Factory\FormFactory service into your controller`, the `use statement` and assigning to variable are kinda self explanatory. 

Unless this is style of whole tutorial, then it is ok.